WordPress/src/main/java/org/wordpress/android/ui/comments/NotificationCommentDetailFragment.kt

```diff
@@ -16,6 +21,9 @@ import org.wordpress.android.util.ToastUtils
  * It'd be better to have multiple fragments for different sources for different purposes
  */
 class NotificationCommentDetailFragment : CommentDetailFragment() {
+    private val note: Note // note will be non-null after onCreate
+        get() = mNote!!
```

Comment:
Wdyt of using this getter for all the `mNote!!` occurrences in this file?

Reply:
Yeah, that makes sense, I missed some occurrences in this file when I made the change.